src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [errorOccured, setErrorOccured] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isEmpty, setIsEmpty] = useState(false);
+  const [selectedPersonSlug, setSelectedPersonSlug] = useState('');
+
+  useEffect(() => {
+    getPeople()
+      .then(response => {
+        setPeopleFromServer(response.map(
+          person => {
+            const mother
+            = response.find(prsn => prsn.name === person.motherName);
+            const father
+            = response.find(prsn => prsn.name === person.fatherName);
+
+            return { ...person, mother, father };
+          },
+        ));
```

Comment:
It looks really messy here. Let's do actions step by step and assign values we receive to a variable. I understand indeed the temptation to provide one-line solution, but making your code clear is usually valued much more that its shorteness

Reply:
also, you don't want to have variables like `prsn` in your code